addons/note/tests/test_note.py

```diff
@@ -1,25 +1,23 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.tests import common
+from odoo.tests.common import TransactionCase
 
-class TestNote(common.TransactionCase):
+
+class TestNote(TransactionCase):
 
     def test_bug_lp_1156215(self):
         """ensure any users can create new users"""
-        cr, uid = self.cr, self.uid
-        IMD = self.registry('ir.model.data')
-        Users = self.registry('res.users')
 
-        _, demo_user = IMD.get_object_reference(cr, uid, 'base', 'user_demo')
-        _, group_id = IMD.get_object_reference(cr, uid, 'base', 'group_erp_manager')
+        demo_user = self.env.ref('base.user_demo')
+        group = self.env.ref('base.group_erp_manager')
```

Comment:
if want id then no need to use self.env.ref you can directly get it using self.ref('..')